### PR TITLE
Multiping

### PR DESCRIPTION
**Pull request by brl0**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: .circleci/build_steps.sh [hunk 1]
```diff
@@ -42,10 +42,10 @@ export CONDA_NPY='19'
 # Make sure build_artifacts is a valid channel
 conda index /home/conda/staged-recipes/build_artifacts
 
-conda install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
+conda install --yes --quiet conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
 export FEEDSTOCK_ROOT="${FEEDSTOCK_ROOT:-/home/conda/staged-recipes}"
 export CI_SUPPORT="${FEEDSTOCK_ROOT}/.ci_support"
-setup_conda_rc "${FEEDSTOCK_ROOT}" "${FEEDSTOCK_ROOT}/recipes" "${CI_SUPPORT}/${CONFIG}.yaml"
+# setup_conda_rc "${FEEDSTOCK_ROOT}" "${FEEDSTOCK_ROOT}/recipes" "${CI_SUPPORT}/${CONFIG}.yaml"
```
Comment (beckermr) [inline .circleci/build_steps.sh]: Please do not edit the CI scripts. These changes need to b backed out.

File: .circleci/run_docker_build.sh [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
 
 REPO_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
+echo $REPO_ROOT
 ARTIFACTS="$REPO_ROOT/build_artifacts"
 THISDIR="$( cd "$( dirname "$0" )" >/dev/null && pwd )"
+echo $THISDIR
 PROVIDER_DIR="$(basename $THISDIR)"
+echo $PROVIDER_DIR
 AZURE="${AZURE:-False}"
+echo $AZURE
```
Comment (beckermr) [inline .circleci/run_docker_build.sh]: Again here do not edit these.

File: recipes/multiping/meta.yaml [hunk 3]
```diff
@@ -3,19 +3,19 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
```
Comment (beckermr) [inline recipes/multiping/meta.yaml]: Please remove the comments in the recipe.

File: recipes/multiping/meta.yaml [hunk 4]
```diff
@@ -3,19 +3,19 @@
...
+{% set name = "multiping" %}
+{% set version = "1.1.12" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/5e/2d/58d80a082f1985c83db0c9927fccb3108cfc6c384d5c55b107a3066eea6b/multiping-1.1.2.tar.gz
```
Comment (beckermr) [inline recipes/multiping/meta.yaml]: you def want the first form of the url.

File: recipes/multiping/meta.yaml [hunk 5]
```diff
@@ -36,7 +36,7 @@ requirements:
   build:
     # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
     # Compilers are named 'c', 'cxx' and 'fortran'.
-    - {{ compiler('c') }}
+    # - {{ compiler('c') }}
```
Comment (beckermr) [inline recipes/multiping/meta.yaml]: you can remove the build section all together.

File: recipes/multiping/meta.yaml [hunk 6]
```diff
@@ -47,35 +47,30 @@ test:
...
+  license: APACHE
   # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
-  license_family: MIT
+  license_family: APACHE
   # It is strongly encouraged to include a license file in the package,
   # (even if the license doesn't require it) using the license_file entry.
   # See https://conda.pydata.org/docs/building/meta-yaml.html#license-file
-  license_file: LICENSE.txt
-  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  license_file: LICENSE
```
Comment (beckermr) [inline recipes/multiping/meta.yaml]: Please make a pr to submit the license file to the sdist upstream.

Comment (brl0) [reply recipes/multiping/meta.yaml]: Thanks for the feedback.
Happy to do it... if I better understood what to do here (sorry, I'm new to this).  I copied this license file directly from the package repo, so I am not sure that I follow where it needs to be submitted.

**Comment by brl0** [comment]
I think I made all requested changes.  Let me know if any additional changes are needed.
Thanks.

File: recipes/multiping/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+{% set version = "1.1.2" %}
+{% set sha256 = "9ecad777ca27c4415cd685957bafaaa0faf4f17f9419800e9f57a298201f2e5d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (beckermr) [inline recipes/multiping/meta.yaml]: Put the sha here directly.

File: recipes/multiping/meta.yaml (hunk 3 above)
Comment (brl0) [reply recipes/multiping/meta.yaml]: Done.

File: recipes/multiping/meta.yaml (hunk 4 above)
Comment (brl0) [reply recipes/multiping/meta.yaml]: Couldn't get the first form to work, but was able to get a similar link to work based on the example link for Github.

File: recipes/multiping/meta.yaml (hunk 6 above)
Comment (brl0) [reply recipes/multiping/meta.yaml]: Done, [here](https://github.com/romana/multi-ping/pull/29).

File: recipes/multiping/meta.yaml (hunk 7 above)
Comment (brl0) [reply recipes/multiping/meta.yaml]: Done.

**Comment by beckermr** [comment]
linux passing - merging

**Pull request merged**